source/styleEditor.js

```diff
@@ -1,3 +1,13 @@
 $(document).ready(function(){
-  // your code goes here.
+
+  $('form').on('submit', function(event){
+     event.preventDefault();
+   });
+
+  $("#style_editor").submit(function() {
+    var input = $(this).serializeArray();
+    $(input[0]['value']).css(input[1]['value'], input[2]['value'])
+  })
```

Comment:
Also, while `.serializeArray()` does work, I'd say readability is at a loss here. I'd suggest using jQuery's .val() method on an input selector just to be more explicit about the values you're pulling from the DOM